docs/use-cases/azure.md

```diff
@@ -0,0 +1,113 @@
+resolve flux reconciliation failures where source-controller is unreachable.
+Using `--network-plugin=azure --network-policy=calico` has been tested to work properly.
+This issue only affects you if you are using `--network-policy` on AKS, which is not a default option.
+
+!!! warning
+    AKS `--network-policy` is currently in Preview
+
+### AAD Pod-Identity
+
+Depending on the features you are interested in using with Flux, you may want to install AAD Pod Identity.
+With [AAD Pod-Identity](https://azure.github.io/aad-pod-identity/docs/), we can create Pods that have their own
+cloud credentials for accessing Azure services like Azure Container Registry(ACR) and Azure Key Vault(AKV).
+
+If you do not use AAD Pod-Identity, you'll need to manage and store Service Principal credentials in K8s Secrets, to integrate Flux
+with other Azure Services.
+
+As a pre-requisite, your cluster must have `--enable-managed-identity` configured.
+
+This software can be [installed via Helm](https://azure.github.io/aad-pod-identity/docs/getting-started/installation/) (unmanaged by Azure).
+Use Flux's `HelmRepository` and `HelmRelease` object to manage the aad-pod-identity installation from a bootstrap repository and keep it up to date.
+
+!!! note
+    As an alternative to Helm, the `--enable-aad-pod-identity` flag for the `az aks create` is currently in Preview.
+    Follow the Azure guide for [Creating an AKS cluster with AAD Pod Identity](https://docs.microsoft.com/en-us/azure/aks/use-azure-ad-pod-identity) if you would like to enable this feature with the Azure CLI.
+
+### Cluster Creation
+
+!!! info
+    When working with the Azure CLI, it can help to set a default `location`, `group`, and `acr`.
+    See `az configure --help`, `az configure --list-defaults`, and `az configure --defaults key=value`
+
+The following creates an AKS cluster with some minimal configuration that will work well with Flux:
+
+```sh
+az aks create \
+ --network-plugin="azure" \
+ --network-policy="calico" \
+ --enable-managed-identity \
+ --enable-pod-identity \
+ --name="my-cluster"
+```
+
+## Flux Installation with Azure DevOps Repos
+
+Ensure you can login to [https://dev.azure.com](dev.azure.com) for your proper organization, and create a new repo to hold your
+flux install and other necessary config.
+
+There is no bootstrap provider currently for Azure DevOps Repos,
+but you can clone your Azure Repo, then use the [Generic Git Server](../guides/installation.md#generic-git-server)
+guide to manually bootstrap Flux. (It must be a Git repo; TFVC Repos are not supported by source-controller)
+Take note of the Azure DevOps specific section within the guide.
+
+If you use the generated SSH deploy key from `flux create source git`, ensure it is an RSA key (not an elliptic curve).
+Make sure you're using the `libgit2` provider, compatible with Git Protocol v2, for any `GitRepository` object
```

Comment:
This line strikes me as a bit confusing and hard to read. Possibly reword to something like

"Ensure you are using the `libgit2` provider for any `GitRepository` object fetching from Azure Repos."

Reply:
Thanks -- reworded this slightly